src/sqs/backfill.test.ts

```diff
@@ -1,18 +1,17 @@
 import { processInstallation } from "../sync/installation";
-import { mocked } from "ts-jest/utils";
+import { mocked } from "jest-mock";
```

Comment:
Mocked in ts-jest/utils is being deprecated and we get warnings all the time so just moving it to the new correct way